extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/convention/homie300/DeviceStatsAttributes.java

```diff
@@ -23,20 +22,7 @@
  */
 @NonNullByDefault
 public class DeviceStatsAttributes extends AbstractMqttAttributeClass {
-    // The following attributes are part of the specification.
-    // They serve limited use though and their location within the device tree is questionable.
-    // We do not require those from the peer device, for now.
-
-    // public int uptime = 0;
-    // public float signal = 0;
-    // public float cputemp = 0;
-    // public float cpuload = 0;
-    // public float battery = 0;
-    // public int freeheap = 0;
-    // public float supply = 0;
-
-    // The interval time is like a heart-beat/keep-alive timer
-    public @MandatoryField int interval = 60; // In seconds
+    public int interval = 0; // In seconds
```

Comment:
is the heartbeat obsolete? or very short?

Reply:
There is the MQTT connection heart-beat already.
And I'm going through the Homie convention with some people at the moment and the $stats node is quite questionable, I rather have a non-breaking "0" default.